maps/src/main/java/com/gluonhq/impl/maps/tile/osm/CachedOsmTileRetriever.java

```diff
@@ -54,11 +56,15 @@
     static boolean hasFileCache;
     static {
         try {
-            File storageRoot = StorageService.create()
-                    .flatMap(StorageService::getPrivateStorage)
-                    .orElseThrow(() -> new IOException("Storage Service is not available"));
-
-            cacheRoot = new File(storageRoot, ".gluonmaps");
+            Optional<File> storageRoot = StorageService.create()
+                    .flatMap(StorageService::getPrivateStorage);
+            if (storageRoot.isEmpty()) {
+                cacheRoot = Files.createTempDirectory(".gluonmaps").toFile();
+                logger.info("StorageService implementation not found.\n" +
+                        "Tiles will be cached at a temporary directory: " + cacheRoot);
+            } else {
+                cacheRoot = new File(storageRoot.get(), ".gluonmaps");
```

Comment:
How about `~/.gluon/maps` ?

Reply:
`.gluonmaps` is all time default location for Maps, if we want to change it, it's worth its own issue and PR, isn't it?

Reply:
ok